lib/EnsEMBL/REST/Model/ga4gh/referenceSets.pm

```diff
@@ -0,0 +1,132 @@
+=head1 LICENSE
+Copyright [1999-2014] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+=cut
+
+package EnsEMBL::REST::Model::ga4gh::referenceSets;
+our @ISA =('EnsEMBL::REST::Model::ga4gh');
```

Comment:
use parent/base is preferrable. This is not how we inherit via Moose, unless the inherited thing is not moosey.